### PR TITLE
Update log bbe and test cases

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: examples/log-api/log_api.bal [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
+    error e = error("something went wrong!");
 
-    // Shows the main log levels that are available. By default, all log messages are logged 
-    // to the console at the `INFO` level. In addition to these log levels, 
-    // there are 2 additional levels named `OFF` and `ALL`.
-    // `OFF` turns off logging and `ALL` enables all the log levels. The log
-    // level can be configured via a Ballerina configuration file or CLI
-    // parameters.
-    log:printDebug("debug log");
+    // There are two log levels (INFO, ERROR) and there is no user configuration to control which log level to print.
```
Comment (praneesha) [inline examples/log-api/log_api.bal:6]:
```suggestion
    // There are two log levels (`INFO` and `ERROR`) and there is no user configuration to control which log level to print.
```

File: examples/log-api/log_api.bal [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
+    // Users can pass any number of key/value pairs which needs to be displayed in the log message.
```
Comment (praneesha) [inline examples/log-api/log_api.bal:7]:
```suggestion
    // Users can pass any number of key/value pairs, which need to be displayed in the log message.
```

File: examples/log-api/log_api.out [hunk 3]
```diff
@@ -1,19 +1,9 @@
-# As shown in the output, only the `INFO` and higher level logs are logged by default.
+# As shown in the output, both `INFO` and `ERROR` level logs are logged.
```
Comment (praneesha) [inline examples/log-api/log_api.out:1]:
```suggestion
# As shown in the output, both the `INFO` and `ERROR` level logs are logged.
```

File: examples/log-api/tests/log_api_test.bal [hunk 4]
```diff
@@ -1,69 +1,35 @@
...
+import ballerina/log;
 
-string printDebug = "";
 string printError = "";
-string printInfo = "";
-string printTrace = "";
-string printWarn = "";
+string print = "";
 
 // This is the mock function which will replace the real function
```
Comment (praneesha) [inline examples/log-api/tests/log_api_test.bal:7]:
```suggestion
// This is the mock function, which will replace the real function.
```

File: examples/log-api/tests/log_api_test.bal [hunk 5]
```diff
@@ -1,69 +1,35 @@
...
+public function mockPrint(string msg, *log:KeyValues keyValues) {
     if (msg is string) {
-        printWarn = msg;
+        print = msg;
     }
 }
 
 @test:Config {}
 function testFunc() {
     // Invoking the main function
```
Comment (praneesha) [inline examples/log-api/tests/log_api_test.bal:31]:
```suggestion
    // Invoking the main function.
```

**Review by praneesha — APPROVED** [review]

**Comment by daneshk** [comment]
@MadhukaHarith92 Shall we add a passing function pointer use case as well?

**Comment by MadhukaHarith92** [comment]
> @MadhukaHarith92 Shall we add a passing function pointer use case as well?

Added in 15559f3

**Comment by MadhukaHarith92** [comment]
Temporarily disabled test cases since passing function pointers to mock functions are not supported at the moment: https://github.com/ballerina-platform/ballerina-lang/issues/27473

**Review by niveathika — APPROVED** [review]

**Pull request merged**